### PR TITLE
Update summary and licensing information; add pageview license

### PR DESCRIPTION
**Pull request by gwicke**
Bug: [T170602](https://phabricator.wikimedia.org/T170602)

### CODE REVIEW

**Comment by Pchelolo** [comment]
Having no idea about the specific licenses, LGTM from the code perspective

**Comment by gwicke** [comment]
LGTM in general to me as well. Lets wait a moment to see if we can generalize this for the other projects as well (see mail thread).

**Comment by d00rman** [comment]
Why is CC0 being added to the mathoid end points as well @marcelrf ?

**Comment by gwicke** [comment]
@d00rman, math is CC-BY-SA 3.0. This is explicit because we just dropped the global license disclaimer. There is a follow-up discussion about maybe re-adding the global disclaimer with something like "Unless otherwise specified in the endpoint documentation, content is licensed under CC-BY-SA 3.0.". We could then consider dropping the explicit mention for the majority of endpoints using that license, and use the same text for other projects.

**Comment by d00rman** [comment]
Oh right right, thnx @gwicke !

File: projects/wikimedia.org.yaml [hunk 1]
```diff
@@ -8,33 +8,44 @@ paths:
             title: Wikimedia REST API
             description: >
                 This API provides cacheable and straightforward access to
-                Wikimedia content and data, in machine-readable formats. Each
-                entry point has explicit stability markers to inform you about
-                development status and change policy, according to [our API
-                version policy](https://www.mediawiki.org/wiki/API_versioning).
+                Wikimedia content and data, in machine-readable formats.
```
Comment (nuria) [inline projects/wikimedia.org.yaml]: Don't we need to add " in machine-readable formats, for the project specified in the URL." according to zhou's comments?

File: projects/wikimedia.org.yaml [hunk 2]
```diff
@@ -8,33 +8,44 @@ paths:
...
-                [Privacy Policy](https://wikimediafoundation.org/wiki/Privacy_policy), and
-                you irrevocably agree to release modifications or additions
-                made through this API under the 
+                [Privacy Policy](https://wikimediafoundation.org/wiki/Privacy_policy).
+                Unless otherwise specified in the endpoint documentation
+                below, content accessed via this API is licensed under the
                 [CC-BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/) 
-                License and the [GFDL](https://www.gnu.org/copyleft/fdl.html).
+                and [GFDL](https://www.gnu.org/copyleft/fdl.html) licenses,
+                and you irrevocably agree to release modifications or
```
Comment (nuria) [inline projects/wikimedia.org.yaml]: I think this needs to be "and you irrevocably agree to release modifications or additions made through this API under the CC-BY-SA 3.0 license and the GFDL."

Comment (gwicke) [inline projects/wikimedia.org.yaml]: I dropped the repetition. I don't think it is needed for clarity.

File: projects/wikimedia.org.yaml (hunk 1 above)
Comment (gwicke) [inline projects/wikimedia.org.yaml]: Some of the data is cross-project as well. For licensing purposes, it shouldn't matter anyway, as differing licenses are called out explicitly.

File: projects/wikimedia.org.yaml (hunk 2 above)
Comment (nuria) [inline projects/wikimedia.org.yaml]: Then, +1 I think you have to merge as i do not have permits correct?

**Pull request merged**